### DISCUSSION TITLE
Add a recycle bin

### DISCUSSION THREAD

**Discussion by JasonHorkles** [pelican-dev/panel#91] [Ideas]
When files are deleted from the panel, it would be nice if there was an option to send them to a recycle bin/trash for some time in case the wrong files were accidentally deleted or something.

**Comment by lancepioch** [comment]
Thanks, we recommend just suspending the server and then deleting it after some time!

**Reply by JasonHorkles** [reply]
I think you misunderstood what I meant. I'm referring to deleting files within the servers, not deleting the servers themselves

**Reply by lancepioch** [reply]
Whoops you are correct that I did misunderstand you.

Unfortunately we cannot do that. I recommend that you set up backups to help you restore older files if they are needed!
